app/views/shared/_manage_volunteers.html.erb

```diff
@@ -0,0 +1,54 @@
+<h1 class="pt-5">Manage Volunteers</h1>
+
+<div id="volunteer-assignment" class="card card-container">
```

Comment:
This ID was only on one of the previous tables, but it was only there for testing and did not apply any styling or functionality. I figured it wouldn't hurt to leave it here so the current tests pass.